*** old/AliESDHeader.cxx	2011-05-26 12:07:05.000000000 +0200
--- new/AliESDHeader.cxx	2011-06-17 10:56:13.855007017 +0200
***************
*** 24,30 ****
  #include "AliTriggerScalersESD.h"
  #include "AliTriggerScalersRecordESD.h"
  #include "AliTriggerIR.h"
! #include "AliTriggerConfiguration.h"
  #include "AliLog.h" 
  
  ClassImp(AliESDHeader)
--- 24,30 ----
  #include "AliTriggerScalersESD.h"
  #include "AliTriggerScalersRecordESD.h"
  #include "AliTriggerIR.h"
! // #include "AliTriggerConfiguration.h"
  #include "AliLog.h" 
  
  ClassImp(AliESDHeader)
